Documentation/Signoffs/Mario Kart Rev 2 - Signoff - RRSA System.md

```diff
@@ -0,0 +1,98 @@
+
+### Analysis:
+
+The Raspberry Pi Zero 2W was found to draw 580 mA (2.9 W) when stressed [1]. The Raspberry Pi Camera draws a maximum of 250 mA according to a forum post by the Principle Software Engineer of the Raspberry Pi Foundation [2]. The internal resistance of the Raspberry Pi's internal GPIO pullup resistor ranges from 33-73 kOhms [3], and with a 3.3 V logic level, the maximum possible current draw owing to the closing of the reed switch is 0.1 mA (calculated by dividing 3.3 V by the lowest specified resistance). The DPS310 altitude sensor requires a maximum current of 0.345 mA [4]. The LEDs in the switches are specified to have a current draw of approximately 20 mA each [5]. The power supply unit is specified to have a nominal efficiency of 90 %. The battery used has an energy storage capacity of 24.42 Wh. Accordingly, the effective energy available to the unit is 21.98 Wh. The required power input to the device may be calculated by multiplying the sum of the above currents with 5 V, since the voltage regulation on the Raspberry Pi board is performed using a linear regulator (meaning the current is constant). This yields a maximum power draw of 4.352 W, corresponding to a battery life of 5.050 hours, which meets the prior specification. The specified battery contains built in circuitry which ensures that it can never be over-discharged nor over-charged. Additionally, the power supply module provides a visual indication of the charge status, lighting up green when the battery is fully charged, and a red warning light when the charge is low. It was for this reason that a clear-topped box was selected. No detailed cycling data was given, making it impossible to make a cycling life estimation; however, the discharge rate is reccomneded to be kept at or below 0.25 C, corresponding to a current of 1.32 A. This device is expected to draw ((4.352 W)/0.9)/(3.7 V) A = 1.31 A nominally, which satisfies the battery's specification. Additionally, the battery is rated for a peak current of 3.3 A. The charge current is limited to 1 A by the selected power supply module, which satisfies the battery's charge current specification of 1.5 A or less. 
+
+The utilized pressure sensor is specified to provide a measurement precision of +/- 2 cm, so it satisfies the prior specification. In addition, it provides on-board measurement averaging and low-pass filtering to reduce noise from the measurement. 
+
+Hammlin specifies that their reed switches can last over 200 million actuations under ideal conditions [6]. The rated current for these reed switches is 0.5 A, which is well above what would be required to provide a signal voltage to the computer. Assuming a 200-million-actuation life and a 26" bike tire, the device would last for 257,732 miles, satisfying the aforementioned specification. 
+
+The selected camera is capable of recording video in HD, 60 fps (or below) and thus satisfies the previously mentioned specification. 
```

Comment:
Is this the complete set of constraints for the camera? What about format? How does the system sync the data to the footage?

Reply:
How does the video get to the PI?